drivers/watchdog/wdt_nrfx.c

```diff
@@ -15,23 +15,34 @@
 
 DEVICE_DECLARE(wdt_nrfx);
 
+/* Macro for calculating watchdog ticks from time in milliseconds.
+ */
+#define WDT_NRF_MS_TO_TICKS(time_ms) ((time_ms * 32768) / 1000)
+
+/* Macro for calculating time in milliseconds from watchdog ticks.
+ */
+#define WDT_NRF_TICKS_TO_MS(ticks) ((ticks * 1000) / 32768)
+
+/* Symbol containing minimum supported timeout time in milliseconds.
+ * Value 0x0F comes from Nordic's Product Specification.
+ */
+#define WDT_NRF_MIN_TIME_MS WDT_NRF_TICKS_TO_MS(0x0FuL)
+
 /* Each activated and not reloaded channel can generate watchdog interrupt.
  * Array m_callbacks provides storing callbacks for each channel.
  */
 static wdt_callback_t m_callbacks[NRF_WDT_CHANNEL_NUMBER];
 
-/*
- * The m_allocated_channels variable stores number of currently allocated
+/* The m_allocated_channels variable stores number of currently allocated
  * and activated watchdog channels.
  */
-static u8_t           m_allocated_channels;
+static u8_t m_allocated_channels;
 
-/*
- * The m_timeout variable stores watchdog timeout value in millisecond units.
+/* The m_timeout variable stores watchdog timeout value in millisecond units.
  * It is used to check whether installing watchdog provide the same timeout
  * value in the window configuration.
  */
-static u32_t          m_timeout;
+static u32_t m_timeout;
 
 
 static void wdt_event_handler(void)
```

Comment:
This could be moved to right before `wdt_init` as it is directly related to this function and used only there.